cis194/week7/daniel-deng/src/CIS194/Homework07/Scrabble.hs

```diff
@@ -1,11 +1,14 @@
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 
-module CIS194.Homework07.Scrabble where
+module CIS194.Homework07.Scrabble ( Score(..), getScore ) where
 
 newtype Score = Score Int
   deriving (Num, Eq, Show, Ord)
 
+getScore :: Score -> Int
+getScore (Score s) = s
```

Comment:
Just FYI. It's conventional to write:
```Haskell
newtype Score = Score { unScore :: Int } 
```
so that `unScore` is available to extract the `Int` out of Score without writing a function like this.

Reply:
Awesome @shineli1984 thanks!